netcore/Makefile

```diff
@@ -51,7 +51,7 @@ TEST_ASSETS_URL = $(FEED_BASE_URL)/corefx-tests/$(NETCORETESTS_VERSION)/$(TESTS_
 BLAME = $(shell git blame ../configure.ac | grep AC_INIT | cut -f1 -d' ')
 VERSTUB = .$(shell git rev-list --count $(BLAME)..HEAD)
 
-all: runtime bcl
+all: bcl
```

Comment:
why `runtime` was removed here?

Reply:
@marek-safar we wanted to be able to build netcore from scratch by just using netcore/build.sh.

Reply:
It was causing circular build dependencies. I don't think we can build with just that script. You need to run `autogen` first and then `make`. Let's stay with this setup.